### DISCUSSION TITLE
Incident with Pages and Actions 9/16/24

### DISCUSSION THREAD

**Discussion by LiteBrite82** [community/community#138852] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

# We are investigating reports of degraded performance for Actions and Pages

https://www.githubstatus.com/incidents/69sb0f8lydp4

**Comment by LiteBrite82** [comment]
Pages seems to be recovering; the build I was working on is back up and running. 

Anyone else having issues with running a deployment?

**Comment by LiteBrite82** [comment, marked as answer]
This incident has been resolved.

**Reply by javiroman** [reply]
It's still not working properly (from Europe):
<img width="2702" height="1215" alt="image" src="https://github.com/user-attachments/assets/adf68a9b-927f-4bf9-997b-fbfc08af45f2" />
